httpfuzzerprocessor/showDifferences.js

```diff
@@ -0,0 +1,76 @@
+//A Fuzzer HTTP Processor script that compares the original Response with the fuzzed Response 
+//and add the result to the state column!
+ 
+var DiffTool = Java.type("org.zaproxy.zap.extension.diff.diff_match_patch");
+var key = "fuzz.httpfuzzer.messageprocessor.tagcreator.tags.text";
+var showResultInTable = true;
+
+
+function processMessage(utils, message) {
+	return message;
+}
+
+// Called after receiving the fuzzed message from the server
+function processResult(utils, fuzzResult){	
+	var original = responseAsString(utils.getOriginalMessage());
+	var fuzzed = responseAsString(fuzzResult.getHttpMessage());
+	var diffList = createDiff(original, fuzzed);	
+	var aggregatedDiff = aggregateDiff(diffList);	
+	displayToStateColumn(fuzzResult, aggregatedDiff);
+	return showResultInTable;
+}
+
+function responseAsString(httpMessage){
+	var responseHeader = httpMessage.getResponseHeader().toString();
+	var responseBody = httpMessage.getResponseBody().toString();
+	return responseHeader + "\r\n" + responseBody;
+}
+
+function createDiff(original, fuzzed){
+	var diffTool = new DiffTool();
+	return diffTool.diff_main(original, fuzzed);
+}
+
+function displayToStateColumn(fuzzResult, aggregatedDiff){
+	removeAllStates(fuzzResult);
```

Comment:
The Problem is, that `fuzzResult.getCustomStates()` can contain many states, but the  FuzzResultStateHighlighter can not display many states. The Payload Reflection Detector is a default Message Processor and writes always to this column, if he detects something. Than the state of this script will not be displayed.  If the first Message Processor render state in the column no other Message Processor can add something.

I add a flag that can control this behaviour, but the flag is `true` by default. So it remove all states by default.